docs/sources/fundamentals/dashboards-overview/index.md

```diff
@@ -0,0 +1,87 @@
+
+## Query
+
+Queries allow for the reduction of data to a specific dataset, providing a more manageable visualization. They help answer questions you have about system and operational processes. For instance, a company with an online store might want to determine the number of customers who add products to their shopping carts. This can be achieved through a query that aggregates access metrics for the shopping cart service, revealing the number of users accessing the service per second.
+
+When working with data sources, it is crucial to recognize that each one has its own distinct query language. For example, Prometheus data sources utilize [PromQL](/blog/2020/02/04/introduction-to-promql-the-prometheus-query-language/), while [LogQL](https://grafana.com/docs/loki/latest/logql/) is used for logs, and particular databases employ SQL. A query supports every visualization in Grafana, and a dashboard might feature a range of query languages.
+
+The following image shows the Query Editor associated with the Prometheus data source. The `node_cpu_seconds_total` query is written in PromQL and requests just one metric.
+
+{{< figure src="/media/docs/grafana/dashboards-overview/example-query.png" max-width="750px" caption="Example Query" >}}
+
+Queries are a very powerful way to precisely express the data you want on a dashboard.
+
+## Transforms
+
+Not all panels will use [transforms](https://grafana.com/docs/grafana/latest/panels-visualizations/query-transform-data/transform-data/). When you are first starting out, you may not need them, but they’re so powerful we have to mention them.  Sometimes the data that comes from your source won’t be shaped correctly.  Common examples include things like:
+
+- You want to combine two fields together, for example, concatenating `Given Name` and `Family Name` into a `Full Name` field
+- You have CSV data (all text), and you want to convert a field types (such as parsing a date or a number out of a string)
+- You want to filter, join, merge, or perform other SQL-like operations that might not be supported by the underlying data source or query language
+
+The following image shows the transform dialog.
+
+Transforms are located next to the **Query** tab in the edit dialog for a panel. Select the transform you want, and define the transform. The following image shows that you can have as many transforms as you want, just like queries. For example, you can chain together a series of transforms that make a change to a data type, filter results, organize columns, and sort the result into one data pipeline. Every time the dashboard is refreshed, the transform applies to the latest data from the data source.
+
```

Comment:
```suggestion

The following image shows the transform dialog.

```

Reply:
Moving from above to this location so that it's positioned right before the image.